docs/valid_values/file.md

```diff
@@ -0,0 +1,63 @@
+List of standard terms for the [Dataset Sharing Plan Model](../model/file.md).
```

Comment:
```suggestion
List of standard terms for the [File Model](../model/file.md).
```

Reply:
Whoops, made a suggestion without realizing the PR has already been merged.

I will file this as a bug instead!